Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -22,12 +22,13 @@ GeneralConfigPane::GeneralConfigPane(wxWindow* parent, wxWindowID id)
 	: wxPanel(parent, id)
 {
 	cpu_cores = {
-		{ 0, _("Interpreter (VERY slow)") },
+		{ PowerPC::CORE_INTERPRETER, _("Interpreter (VERY slow)") },
+		{ PowerPC::CORE_JITGENERIC, _("JIT Generic (slow)") },
 #ifdef _M_X86_64
-		{ 1, _("JIT Recompiler (recommended)") },
-		{ 2, _("JITIL Recompiler (slower, experimental)") },
+		{ PowerPC::CORE_JIT64, _("JIT Recompiler (recommended)") },
+		{ PowerPC::CORE_JITIL64, _("JITIL Recompiler (slower, experimental)") },
```

Comment:
It's misleading that JITIL says "slower" while the option that this PR adds says "slow". Could we make JIT say "fastest, recommended" and JITIL only "experimental"?